trainer/derrian_backend/custom_scheduler/LoraEasyCustomOptimizer/low_bit_optim/subclass_fp8.py

```
@@ -0,0 +1,198 @@
+from torch.utils._python_dispatch import return_and_correct_aliasing
+
+from torchao.utils import TORCH_VERSION_AT_LEAST_2_5, TorchAOBaseTensor
+
+aten = torch.ops.aten
+c10d_functional = torch.ops.c10d_functional
+_c10d_functional = torch.ops._c10d_functional
+
+DTYPE = torch.float8_e4m3fn
+
+
+def quantize_fp8(input: Tensor, block_size: int):
```

Ruff (0.15.13)

[error] Function argument `input` is shadowing a Python builtin

(A002)